checkstyle-tester/launch.groovy

```diff
@@ -207,13 +206,20 @@ def runMavenExecution(srcDir, excludes, checkstyleConfig) {
     println "Running Checkstyle on $srcDir - finished"
 }
 
-def postProcessCheckstyleReport(siteDir) {
+def postProcessCheckstyleReport(targetDir) {
+    def siteDir = "$targetDir/site"
     println 'linking report to index.html'
     new File("$siteDir/index.html").renameTo "$siteDir/_index.html"
     Files.createLink(Paths.get("$siteDir/index.html"), Paths.get("$siteDir/checkstyle.html"))
 
     removeNonReferencedXrefFiles(siteDir)
     removeEmptyDirectories(new File("$siteDir/xref"))
+
+    new AntBuilder().replace(
+        file: "$targetDir/checkstyle-result.xml",
+        token: "src/main/java",
```

Comment:
@rnveach 
Fixed.
Now it replaces all occurrences of 'checkstyle-tester/src/main/java' with 'checkstyle-tester/repositories'